search-provider/applicationV1.js

```diff
@@ -0,0 +1,30 @@
+const Lang = imports.lang;
```

Comment:
Well that is the file that was originally called searchProviderV1.js before the pull request. Could squash them into one file, partially just laziness here. And seeing as I plan to rewrite this to C as soon as this patch set lands probably not a big deal.